Added FormatStackTraceAsArray option to write StackTrace as an array

oscarmorasu opened the pull request:
**What issue does this PR address?**
This PR fixes #229

**Does this PR introduce a breaking change?**
No. The new flag is false by default, which keeps the current behavior. If the flag is set to true the new behavior takes place.

**Please check if the PR fulfills these requirements**
- [X] The commit follows our [guidelines](https://github.com/serilog/serilog/blob/dev/CONTRIBUTING.md)
- [ ] Unit Tests for the changes have been added (for bug fixes / features)

**Other information**:

mivano commented:
Thanks for the PR. Would it be possible to add a unit test for this? And can you also list this option in the readme.md to explain to users why you want this option?

oscarmorasu commented:
@mivano, I added two unit tests.

Also, I changed the [Configure the Sink](https://github.com/serilog/serilog-sinks-elasticsearch/wiki/Configure-the-sink) wiki page, explaining the use of the flag. The README.md doesn't seem to have any content related to the Sink Options.

mivano commented:
Nice, thanks. I merged it into dev, so beta packages should be available soon.
